app/components/bali/avatar/component.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= tag.div **@options do %>
+  <figure class="image is-128x128">
+    <% if picture? %>
+      <%= picture %>
+    <% else %>
+      <img class="is-rounded" src="<%= @placeholder_url %>" data-image-preview-target="output">
+    <% end %>
+  </figure>
+  <div class="icon-content">
+    <label for="file-input">
+      <%= render Bali::Icon::Component.new('camera') %>
+    </label>
+    <div class="control">
+      <%= @form.file_field @attr_model_name, accept: @accepted_formats, id: 'file-input',
```

Comment:
Should generate an ID specific for this field, otherwise if we add multiple Avatar components in a page we will have invalid HTML since ID's should be unique.

```suggestion
      <%= @form.file_field @attr_model_name, accept: @accepted_formats,
```